extension/page-detect.js

```diff
@@ -43,6 +43,8 @@ window.pageDetect = (() => {
 
 	const isNotifications = () => /\/notifications(\/participating)?/.test(location.pathname);
 
+	const isSettings = () => isRepo() && /^\/settings/.test(getRepoPath());
```

Comment:
`isRepoSettings`, as `isSettings` is too vague. It could be user account settings.